webapp/platform/types/src/posts.ts

```diff
@@ -231,3 +233,55 @@ export type NotificationResult = {
     reason?: string;
     data?: string;
 }
+
+export type MessageData = {
```

Comment:
Could you actually move `MessageData` and `UserActivityProp` to `mattermost-redux/src/utils/post_list`? These are only used for synthetic posts made by the web app, so we don't need them to be in the public package